components/support/rc_crypto/nss/nss_sys/src/bindings/pk11pub.rs

```diff
@@ -9,6 +9,9 @@ extern "C" {
     pub fn PK11_FreeSlot(slot: *mut PK11SlotInfo);
     pub fn PK11_GetInternalSlot() -> *mut PK11SlotInfo;
     pub fn PK11_GetInternalKeySlot() -> *mut PK11SlotInfo;
+    pub fn PK11_NeedUserInit(slot: *mut PK11SlotInfo) -> PRBool;
+    pub fn PK11_NeedLogin(slot: *mut PK11SlotInfo) -> PRBool;
+    pub fn PK11_CheckUserPassword(slot: *mut PK11SlotInfo, password: *const c_char) -> SECStatus;
```

Comment:
@bendk, do we have a guideline on how to sort these methods?

Reply:
I don't think it matters, but @mhammond would know best.

Reply:
I don't think we have any such guidelines. I think the general story is "if clippy is happy, we are happy" :)